qmd/index.qmd

```diff
@@ -1,6 +1,6 @@
 # Introduction {.unnumbered}
 
-> Make collecting and sharing high-quality, interoperable environmental surveillance data as easy as possible.
+q> Facilitating the Collection and Sharing of High-Quality, Interoperable Environmental Surveillance Data: A Community Open Science Project. 
```

Comment:
I am not sure what "q>" means but it is being rendered as "q>". If this is intentional feel free to disregard.

Reply:
This was a typo. Resolved.